client/errors.go

```diff
@@ -142,6 +142,9 @@ const (
 
 	// ErrorKindValidationSetNotFound: validation set cannot be found.
 	ErrorKindValidationSetNotFound ErrorKind = "validation-set-not-found"
+
+	// ErrorKindBusySnap: cannot do action as snap is currently busy.
+	ErrorKindBusySnap ErrorKind = "busy"
```

Comment:
I'd tweak the value, extend a comment a little bit, and group it with other ErrorKindSnap above.
```suggestion
	// ErrorKindSnapBusy: cannot do action as snap has applications running 
	// and is  considered to be "busy". This is a temporary error. The error
	// should be distinguished from "snap-change-conflict", which is raised
	// when there is an ongoing, conflicting change targeting a given snap. 
	ErrorKindSnapBusy ErrorKind = "snap-busy"
```